examples/load_from_cloud.html

```diff
@@ -0,0 +1,442 @@
+        const PATH = document.getElementById("path").value;
+        const REGION = document.getElementById("aws_region").value;
+        const BUCKET = document.getElementById("aws_bucket").value;
+        const FOLDER = document.getElementById("aws_folder").value;
+        const AWS_ACCESS_KEY_ID = document.getElementById("aws_key").value;
+        const AWS_SECRET_ACCESS_KEY = document.getElementById("aws_secret").value;
+        document.getElementById("login-form").style.display = "none";
+        document.getElementById("potree-container").style.display = "initial";
+    		window.viewer = new Potree.Viewer(document.getElementById("potree_render_area"));
+
+    		viewer.setEDLEnabled(true);
+    		viewer.setFOV(60);
+    		viewer.setPointBudget(1_000_000);
+    		viewer.loadSettingsFromURL();
+
+    		viewer.setDescription("");
+
+    		viewer.loadGUI(() => {
+    			viewer.setLanguage('en');
+    			$("#menu_appearance").next().show();
+    			//viewer.toggleSidebar();
+    		});
+
+        const s3 = new AWS.S3({
+          region: REGION,
+          accessKeyId: AWS_ACCESS_KEY_ID,
+          secretAccessKey: AWS_SECRET_ACCESS_KEY
+        });
+
+        const signUrl =
+          // For AWS JavaScript API v2, this doesn't need to be async.
+          // But v3 and other cloud APIs will need async / await.
+           async (url, headers) => s3.getSignedUrl('getObject',
+                                            {
+                                              Bucket: BUCKET,
+                                              Key: url,
+                                              ...(headers || {})
+                                            });
+
+    	    const e = await Potree.loadPointCloud(FOLDER + "/" + PATH, NAME, null, signUrl);
+    	    viewer.scene.addPointCloud(e.pointcloud);
+    	    const material = e.pointcloud.material;
+    	    material.size = 1;
+    	    material.pointSizeType = Potree.PointSizeType.ADAPTIVE;
+
+    	    viewer.fitToScreen();
+       }
+
+    const awsButton = document.getElementById("aws-start-button");
+    awsButton.addEventListener('click', runAWSExample);
+
+
+  // Microsoft Azure
+
+  // generate key object for signing algorithm
+  async function getAzureKey(key) {
+    const format = "raw";
+    const keyData = base64StringToArrayBuffer(key);
+    const algorithm = {
+      name: "HMAC",
+      hash: "SHA-256",
+    };
+    const extractable = true;
+    const keyUsages = ['sign','verify'];
+    const result = crypto.subtle.importKey(
+    format,
+    keyData,
+    algorithm,
+    extractable,
+    keyUsages
+    );
+    return result;
+  }
+
+  // hmac signing algorithm
+  async function hmacSHA256(key, data) {
+    const algorithm = "HMAC";
+    const keyObj = await getAzureKey(key);
+    var enc = new TextEncoder(); // always utf-8
+    const encodedBufferArray = enc.encode(data);
+    const signature = crypto.subtle.sign(algorithm, keyObj, encodedBufferArray);
+    return signature;
```

Comment:
I don't think this is a "signature". I think it's the signed data. But I wouldn't worry about naming this thing. Just return it:
```
return crypto.subtle.sign(algorithm, keyObj, encodedBufferArray);
```